gramine-sealing-key-provider.manifest.template

```diff
@@ -39,8 +40,8 @@ sgx.trusted_files = [
   "file:{{ arch_libdir }}/",
 ]
 
-sgx.thread_num = 32
+# sgx.thread_num = 32
 sgx.max_threads = {{ '1' if env.get('EDMM', '0') == '1' else '32' }}
 
 # Enable enclave measurement for sealing
-sgx.enable_stats = true
+sgx.enable_stats = false
```

Comment:
this has changed to be conditional in the most recent commit. Maybe you could rebase to head and it will be fixed.